src/Evolution/DgSubcell/TciStatus.hpp

```diff
@@ -23,6 +23,18 @@ namespace evolution::dg::subcell {
 // @{
 /// Set `status` to `1` if the cell is marked as needing subcell, otherwise set
 /// `status` to `0`. `status` has grid points on the currently active mesh.
+///
+/// \note It is possible to encounter a `status` of `0`, indicating a smooth
+/// solution, even though the active mesh is the subcell mesh. This can occur
+/// when using a multistep time integration method like Adams Bashforth, where
+/// the FD/FV scheme is used as long as _any_ of the timestepper history is
+/// flagged for FD/FV subcell evolution. An example of this would be just after
+/// a shock leaves the cell: the solution is now smooth (`status` takes value
+/// `0`), but the multistep method continues to take FD timesteps (`status` is
+/// defined on the subcell mesh) until the entire history is flagged as smooth.
+/// Thus, he `tci_history.front()` is used to set the status when using
```

Comment:
I would suggest `Thus tci_history.front() (which corresponds to the most recent tci status) is used...`